projects/core/src/datetimepicker/calendar-body.scss

```diff
@@ -55,7 +55,7 @@ $mat-calendar-body-cell-content-size: 100% - $mat-calendar-body-cell-content-mar
   height: $mat-calendar-body-cell-content-size;
   color: rgba(black, 0.87);
   border: $mat-calendar-body-cell-content-border-width solid transparent;
-  border-radius: 50%;
+  border-radius: 50px;
```

Comment:
`px` is not a reliable unit..

Reply:
For edge radius px it is preferred over other units because if the element is a rectangle, it makes sure it looks like a rectangle with perfect border radius, but with% it will look like a diamond with border radius, don't worry, I have already ensured that the change does not affect the actual look and feel of the components.

Reply:
ah my bad. getting sloppy..^^